.husky/pre-commit

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
```

Comment:
This is the pre-commit hook that runs linting and formatting. It will error if there is an error that can't be autofixed.